src/sentry/ingest/consumer/processors.py

```diff
@@ -87,7 +87,13 @@ def process_event(
     # keeping it around because it does provide some protection against
     # reprocessing good events if a single consumer is in a restart loop.
     deduplication_key = f"ev:{project_id}:{event_id}"
-    if cache.get(deduplication_key) is not None:
+
+    try:
+        cached_value = cache.get(deduplication_key)
+    except Exception as exc:
+        raise Retriable(exc)
```

Comment:
There's additional calls to redis and celery below (`event_processing_store.store(data)`, `task.delay` in `submit_symbolicate`). Should these also be marked as `Retriable`?

Reply:
They should already be wrapped in Retriable -- did this in an earlier PR. Just spotted this one I forgot earlier.